src/util.ts

```diff
@@ -423,11 +351,44 @@ export class Util {
         }
         return haveLevel >= requiredLevel;
     }
-}
 
-interface IUploadResult {
-    mxcUrl: string;
-    size: number;
+    public static GetUrlFromMxc(mxc: string, homeserverUrl: string, width: number = 0,
```

Comment:
doesn't the bot sdk have a function for this? Recalling something like this somehow

Reply:
Yeah I remember Travis saying so.

Reply:
nvm soru just noticed she did this manually, too, so it's probably fine

Reply:
Funnily enough, we weren't using it at all. I think we *did* switch to the SDKs methods.